option.go

```diff
@@ -401,9 +402,10 @@ func GrpcRunner(name string, cc *grpc.ClientConn) Option {
 		}
 		delete(bk.runnerErrs, name)
 		r := &grpcRunner{
-			name: name,
-			cc:   cc,
-			mds:  map[string]protoreflect.MethodDescriptor{},
+			name:            name,
+			cc:              cc,
+			mds:             map[string]protoreflect.MethodDescriptor{},
+			traceHeaderName: strings.ToLower(defaultTraceHeaderName),
```

Comment:
Same as https://github.com/k1LoW/runn/pull/742#discussion_r1441145862

```suggestion
			traceHeaderName: defaultTraceHeaderName,
```